packages/graphql-server/src/types.ts

```diff
@@ -75,6 +75,12 @@ export interface RedwoodGraphQLContext {
   [index: string]: unknown
 }
 
+export interface RedwoodOpenTelemetryConfig {
```

Comment:
Could we perhaps have some inline comments to describe what enabling each does?

for example, setting resolvers to true adds ... ?

This?

```
          if (options.resolvers) {
             extendContext({
               [tracingSpanSymbol]: executionSpan,
```

Adds execution span telemetry for each graph resolver?

Reply:
Added the documentation to the options